Pcap++/CMakeLists.txt

```diff
@@ -21,6 +18,18 @@ add_library(
   # Force light pcapng to be link fully static
   $<TARGET_OBJECTS:light_pcapng>)
 
+if(DPDK_VERSION VERSION_LESS "23.11")
```

Comment:
I would say the same keep this MR small and we will add this when 23.11 is realeased